src/main/scala/org/globalforestwatch/layers/PeatlandsFlux.scala

```diff
@@ -1,9 +0,0 @@
-package org.globalforestwatch.layers
-
-import org.globalforestwatch.grids.GridTile
-
-case class PeatlandsFlux(gridTile: GridTile)
-    extends BooleanLayer
-    with OptionalILayer {
-  val uri: String = s"$basePath/peatlands_flux/v20190816/raster/epsg-4326/${gridTile.gridSize}/${gridTile.rowCount}/is/geotiff/${gridTile.tileId}.tif"
```

Comment:
Change path to "$basePath/gfw_peatlands_extent_flux_model/v20190816/raster/epsg-4326/${gridTile.gridSize}/${gridTile.rowCount}/is/geotiff/${gridTile.tileId}.tif"